src/main/webapp/app/usermanagement/research-group/research-group-info/research-group-info.component.html

```diff
@@ -0,0 +1,119 @@
+              [control]="form.controls.school"
+              [model]="form.controls.school.value || ''"
+              placeholder="{{ 'researchGroup.groupInfo.placeholders.school' | translate }}"
+            />
+
+            <jhi-string-input
+              [id]="'rg-defaultFieldOfStudies'"
+              label="{{ 'researchGroup.groupInfo.fields.defaultFieldOfStudies' | translate }}"
+              [control]="form.controls.defaultFieldOfStudies"
+              [model]="form.controls.defaultFieldOfStudies.value || ''"
+              placeholder="{{ 'researchGroup.groupInfo.placeholders.defaultFieldOfStudies' | translate }}"
+            />
+
+            <jhi-string-input
+              [id]="'rg-website'"
+              label="{{ 'researchGroup.groupInfo.fields.website' | translate }}"
+              [control]="form.controls.website"
+              [model]="form.controls.website.value || ''"
+              placeholder="{{ 'researchGroup.groupInfo.placeholders.website' | translate }}"
+            />
+
+            <jhi-string-input
+              [id]="'rg-email'"
+              label="{{ 'researchGroup.groupInfo.fields.email' | translate }}"
+              [control]="form.controls.email"
+              [model]="form.controls.email.value || ''"
+              placeholder="{{ 'researchGroup.groupInfo.placeholders.email' | translate }}"
+            />
+
+            <!-- Address section -->
+            <jhi-string-input
+              [id]="'rg-city'"
+              label="{{ 'researchGroup.groupInfo.fields.city' | translate }}"
+              [control]="form.controls.city"
+              [model]="form.controls.city.value || ''"
+              placeholder="{{ 'researchGroup.groupInfo.placeholders.city' | translate }}"
+            />
+            <jhi-string-input
+              [id]="'rg-postalCode'"
+              label="{{ 'researchGroup.groupInfo.fields.postalCode' | translate }}"
+              [control]="form.controls.postalCode"
+              [model]="form.controls.postalCode.value || ''"
+              placeholder="{{ 'researchGroup.groupInfo.placeholders.postalCode' | translate }}"
+            />
+
+            <jhi-string-input
+              [id]="'rg-address'"
+              label="{{ 'researchGroup.groupInfo.fields.address' | translate }}"
+              [control]="form.controls.address"
+              [model]="form.controls.address.value || ''"
+              placeholder="{{ 'researchGroup.groupInfo.placeholders.address' | translate }}"
+            />
+            <div></div>
+          </div>
+
+          @if (hasInitialized()) {
+            <label class="block mb-2" [for]="'rg-description'">{{ 'researchGroup.groupInfo.fields.description' | translate }}</label>
+            <jhi-editor
+              [id]="'rg-description'"
+              [characterLimit]="1000"
+              [control]="form.controls.description"
+              placeholder="{{ 'researchGroup.groupInfo.placeholders.description' | translate }}"
```

Comment:
the jhi-editor component supports translation, you can set the [shouldTranslate] flag to true and then use the placeholder key directly instead of using translation pipe
```suggestion
              placeholder='researchGroup.groupInfo.placeholders.description'
              [shouldTranslate]="true"
```